src/test/java/org/openstreetmap/atlas/checks/validation/intersections/SelfIntersectingPolyLineCheckTest.java

```diff
@@ -119,4 +123,11 @@ public void testInvalidAreaBuildTag()
         this.verifier.actual(this.setup.getInvalidAreaBuildingTag(), check);
         this.verifier.verifyExpectedSize(1);
     }
+
+    @Test
+    public void testMinimumHighwayType()
```

Comment:
This checks that the filter works properly for values greater than the configurable. Can we get one that checks that values below the configurable get ignored?